NEWS.md

```diff
@@ -70,6 +70,9 @@
 
 * `literal_coercion_linter()` reports a replacement in the lint message, e.g. code like `as.integer(1)` will
   suggest using `1L` instead, and code like `as.numeric(NA)` will suggest using `NA_real_` instead (#1439, @MichaelChirico)
+  
+* Added check for `.lintr` configuration file in the `.github/linters` project sub folder. This is to improve 
```

Comment:
This is a change in behavior and thus potentially a breaking change: previously if a repo contained `.github/linters/.lintr` and no other lintr configs and a local `~/.lintr` is present, lintr 3.0.2 would pull the one from $HOME instead of the one in `.github`. Please move this to a new "breaking changes" section.
Also not sure about the wording @MichaelChirico any thoughts from you as a native?

Reply:
I've updated the file from the pointer above, also changed the wording, not sure if I should mark this as resolved